src/window.rs

```diff
@@ -51,6 +106,9 @@ fn stringify_message_content(message: &TelegramMessage, chat: &Chat) -> String {
                 }
             }
         },
+        MessageContent::MessageContactRegistered { .. } => {
```

Comment:
```suggestion
        MessageContent::MessageContactRegistered => {
```

The variant has no fields, so it's not needed.

Reply:
Also resolved in `chat_row.rs`